pkg/analytics/analytics.go

```diff
@@ -60,7 +60,7 @@ func SendAnonymouscmdInfo() {
 			ApplicationName:    "testkube",
 			ApplicationVersion: commands.Version,
 		},
-		CustomDimensions: gatypes.StringList(os.Args),
+		CustomDimensions: gatypes.StringList(os.Args[1:]),
```

Comment:
Do you check, that slice has more than 1 element?

Reply:
Done ✅